### DISCUSSION TITLE
error after latest update of local REST API

### DISCUSSION THREAD

**Discussion by ygol** [coddingtonbear/obsidian-local-rest-api#94] [General]
### Describe the bug

It seems to be broken:
```
Error
Could not send content to Obsidian: (Error Code 50000) Internal Server Error Cannot read properties of undefined (reading 'getDailyNoteSettings')
```

Anyone else experiencing the same?

### To Reproduce

1. Go to a website page
2. Click on the browser's add-on icon
3. choose template "Append to current monthly note"

### Operating System

windows 11

### Browser

Vivaldi

### Browser Version

6.5.3206.48 (Stable channel) (64-bit)

**Comment by coddingtonbear** [comment]
Duplicate of #93
